tweepy/streaming.py

```diff
@@ -78,8 +78,17 @@ def on_data(self, raw_data):
         elif 'warning' in data:
             if self.on_warning(data['warning']) is False:
                 return False
+        elif 'scrub_geo' in data:
+            if self.on_scrub_geo(data['scrub_geo']) is False:
+                return False
+        elif 'status_withheld' in data:
+            if self.on_status_withheld(data['status_withheld']) is False:
+                return False
+        elif 'user_withheld' in data:
+            if self.on_user_withheld(data['user_withheld']) is False:
+                return False
         else:
-            logging.error("Unknown message type: " + str(raw_data))
+            logging.error("Unknown message type: %s", str(raw_data))
```

Comment:
I think you could just write:

```python
logging.error("Unknown message type: %s", raw_data)
```

Since `%s` implies `str()`

Reply:
Resolved by https://github.com/tweepy/tweepy/commit/cf172c6923cb019734a10f54f8a2d530658326a8